Get_a_name/Deberdelsemáforo/main.js

```diff
@@ -0,0 +1,24 @@
+document.getElementById('stopButton').onclick = illuminateRed;
+document.getElementById('slowButton').onclick = illuminateYellow;
+document.getElementById('goButton').onclick = illuminateGreen;
+
+function illuminateRed() {
+  clearLights();
+  document.getElementById('stopLight').style.backgroundColor = "red";
+}
+
+function illuminateYellow() {
+  clearLights();
+  document.getElementById('slowLight').style.backgroundColor = "yellow";
+}
+
+function illuminateGreen() {
+	clearLights();
```

Comment:
this indentation should be consistent with that of the other functions in this file